src/Classes/Particles.ts

```diff
@@ -57,7 +57,7 @@ export class Particles {
             p.update(i, delta);
 
             /* interaction auto between particles */
-            if (options.particles.lineLinked.enable || options.particles.move.attract.enable) {
+            if (options.particles.lineLinked.enable || options.particles.move.attract.enable || options.particles.move.bounce) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139697)